src/gengowatcher/main.py

```
@@ -399,6 +402,10 @@ def start_web_server():
     except Exception as e:
         log.exception("UI loop crashed")
     finally:
+        try:
+            stats_manager.end_session()
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)